exec/java-exec/src/main/java/org/apache/drill/exec/store/dfs/easy/EasyFormatPlugin.java

```diff
@@ -126,8 +127,12 @@ CloseableRecordBatch getReaderBatch(FragmentContext context, EasySubScan scan) t
     final ImplicitColumnExplorer columnExplorer = new ImplicitColumnExplorer(context, scan.getColumns());
 
     if (!columnExplorer.isSelectAllColumns()) {
+      // We must make sure to pass a table column (not to be confused with implicit column) to the underlying record reader.
+      List<SchemaPath> tableColumns =
```

Comment:
In original PR  I have created helper class which contained common logic for parquet and test format plugins. Somehow I missed that this part is unique for text format plugin, and should NOT be used in parquet one. That's why I have removed it from ImplicitColumnExplorer and added to EasyFormatPlugin.